static/src/javascripts/projects/common/modules/identity/api.ts

```diff
@@ -249,6 +249,11 @@ export const getUserFromApiWithRefreshedCookie = (): Promise<unknown> => {
 	}).then((resp) => resp.json());
 };
 
+export const refreshOktaSession = (returnUrl: string): void => {
+	const endpoint = `${profileRoot}/signin/refresh?useOkta=true&returnUrl=${returnUrl}`;
+	window.location.replace(endpoint);
```

Comment:
Do we need any error handling for the URL that's constructed - especially if the returnURL is invalid for any reason? Or is that handled elsewhere?

Reply:
So we validate the `returnUrl` in gateway here: https://github.com/guardian/gateway/blob/c003028dcef6acab4ded8aab2987f0590cc166f9/src/server/lib/validateUrl.ts#L14

has to be on a guardian domain, with specific path and https